gorush/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Gorush container
+
+# Stage1: build from source
+FROM quay.io/cybozu/golang:1.12-bionic AS build
```

Comment:
Currently, we are using the latest golang version 1.13.
Could you update it?

Reply:
Thank you for your review.
I'll update it.

Reply:
Done